src/Resources/Locales/en.json

```diff
@@ -671,6 +673,13 @@
                         "information": "When there are too many datapoints to display, values will be grouped in the chart. Choose the aggregation method for these groupings."
                     }
                 }
+            },
+            "errors": {
+                "genericNetwork": "Cluster url is invalid or network error occured.",
```

Comment:
Did we do a review on these with @rinisbet? They mostly look fine but I notice we don't ever give them direction on how to resolve the issue.

Reply:
They will do content pass on these errors. We put learn more link which points to the data history documentation. Not sure how much granular documentation we need here, I am sure @rinisbet will share his feedback on that.